server/sonar-web/src/main/js/app/components/embed-docs-modal/EmbedDocsPopup.tsx

```diff
@@ -97,13 +111,31 @@ export default class EmbedDocsPopup extends React.PureComponent<Props, State> {
     );
   }
 
+  getZohoDeskUrl() {
+    const { currentUser } = this.props;
+    let payLoad = {
+      "operation": "signup",
+      "email": currentUser.email,
+      "loginName": "support.autorabit",
+      "fullName": currentUser.name,
+      "utype": "portal",
+      "remoteauthkey": "a0699be8e3f85503a74f7622c7e1f05ba94f7594a265d0899ff81ec962197b66632fbc6aaafb834be8690f3fb7476acecfdf86a5ceecf232f3968aeb35344724",
```

Comment:
The **remoteauthkey** should be set on backend. That's a secure key which no one should know. Read it via EnvUtils on backend.